### PR TITLE
Add more QuestionSettings stories

### PR DESCRIPTION
**Pull request by oisincoveney**
Add more QuestionSettings stories

### CODE REVIEW

**Comment by oisincoveney** [comment]
This change is part of the following stack:

- #51110 ◀

<sub>Change managed by [git-spice](https://abhinav.github.io/git-spice/).</sub>

**Comment by oisincoveney** [comment]
@metabase-bot backport release-x.51.x

File: enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/util.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import type { ReactNode } from "react";
```
Comment (npretto) [inline enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/util.tsx:1]: This file here looks a bit mis-named/mis-placed but I can see why it's hard to give it a name/place.

I have an alternative: [add sdk theme selector to storybook](https://github.com/metabase/metabase/pull/51122)

**Comment by oisincoveney** [comment]
Closing in favor of https://github.com/metabase/metabase/pull/51122

**Pull request closed without merging**